### ISSUE TITLE
hint distribution settings

### ISSUE THREAD

**Issue by briaguya-ai** [briaguya-ai/rando-issue-tracker#62]
it'd be good to at least document what the hint settings for 3drando are compared to n64 rando

![image](https://user-images.githubusercontent.com/70942617/176901658-34c2d0ca-066f-421b-91a6-76b0255b3199.png)

i know fuzzy wanted to use tournament hint settings but i'm not sure if 3drando has that option, so maybe splitting this out to create new hint distribution settings makes sense too

**Comment by Flax4** [comment]
3ds rando has: Balanced, Strong, Very Strong and Useless

**Comment by briaguya-ai** [comment]
> 3ds rando has: Balanced, Strong, Very Strong and Useless

yes but what do those mean? "Tournament" is very clear as to what the hint settings are. "Many powerful hints" is pretty vague.

**Comment by briaguya-ai** [comment]
marking this as "not v1", don't want to mess with the 3drando logic for hints yet

**Comment by Sautros** [comment]
[Hint test spoil logs.zip](https://github.com/briaguya-ai/rando-issue-tracker/files/9058202/Hint.test.spoil.logs.zip)
[Hint distribution tables.xlsx](https://github.com/briaguya-ai/rando-issue-tracker/files/9058203/Hint.distribution.tables.xlsx)


files attached above containing my results table and the original JSONs for reference.

Setting all settings to open, 6 medallions required for rainbow bridge. 

I generated 15 logs and counted the individual WOTH hints, foolish choice hints (which IMO are just as powerful as WOTH), junk/joke hints, and then others.

Based on the table results in the excel doc, 'balanced' seems to produce between 5-6 WOTH hints with anywhere between 3-6 junk hints.

changing the seed gen to 'strong hints' removes ALL junk hints from the pool and increases the WOTH hints to between 7-9. Foolish choice hints seem to be unaffected, sticking at around 1-3 for both categories.

selecting Very Strong launches the WOTH hint numbers between 12-14, keeps junk hints at zero, and in two cases also provided a noticeable increase in 'foolish choice' hints, though I have not combed for duplicates and this could also just be impacted by the item placement in the seed itself. it's difficult to conclude with a small sample size
